pythonsrc/parser/main.py

```diff
@@ -69,9 +74,21 @@ def get_column_lineage(query: str, schema: dict, dialect: str):
         return {"columns": []}
 
     result = []
+    conditions = []
+
+    conditional_cols = extract_query(optimized.find(exp.Where))
+    
+    
+	for col in conditional_cols:
+		col_name = str(str(col).split(".")[1]) if len(str(col).split(".")) == 2 else str(str(col).split(".")[0])
```

Comment:
what happens in different scenarios? e.g.
- `database.schema.table`
- `schema.table`
- `table`